taipy/core/_core.py

```diff
@@ -84,20 +86,37 @@ def stop(self, wait: bool = True, timeout: Optional[float] = None):
         with self.__class__.__lock_is_running:
             self.__class__._is_running = False
 
-    @staticmethod
-    def __update_core_section():
+        with self.__class__.__lock_version_is_initialized:
+            self.__class__._version_is_initialized = False
```

Comment:
This is to unblock the configuration update, mainly for Notebook use-case, where the user wants to add or modify the Config after running the app for a while for testing.

We of course can only `Config.unblock` here, but then when the Core service start again, the version will not be manage. This can be good or bad base on the use-case. 

So the question here is that when re-start the Core service (in notebook), do we want to:
- clean all entities (manage version)
- keep all entities (only unblock the config)